### DISCUSSION TITLE
Independent usage of multiple Room Assistant instances

### DISCUSSION THREAD

**Discussion by andreypolyak** [mKeRix/room-assistant#482] [Q&A]
Hi all,
I am using Room Assistant on one Raspberry Pi Zero and it worked great. Really awesome project!
Recently I bought a second Raspberry and want to use Room Assistant on it as well. But the problem is that I want to use two Room Assistant instances independently. I.e. I want to have two independent device trackers for each tracked device -- one for each Raspberry.

Now, as far as I understand, by default Room Assistant creates a cluster if multiple instances are running on the same subnet, but this behavior can be changed by setting ```autoDiscovery``` property to ```false```. But I found another problem. Room Assistant uses the same MQTT topic for each tracked device (```room-assistant/device_tracker/ble-(device id here)-tracker```). Is it possible to either change the base topic (use for example ```room-assistant2``` instead of ```room-assistant```) or to change the device name (use ```ble-(custom name here)-tracker``` instead of ```ble-(device id here)-tracker```)?

Thanks!

**Comment by mKeRix** [comment]
In the current version neither of this is possible, but it's not a huge change and something that I could probably implement quite soon. Changing the name should be covered by #433 - would that be enough for your use case? If not please open a feature request to allow changing the base topics, so that we can track this better.

**Reply by andreypolyak** [reply]
Hi,
Thanks for the response!
Yes, #433 will cover my use case. I assume it will work both for BLE and BT Classic integration, right?

Regards,
Andrey

**Reply by mKeRix** [reply]
That would be my plan, yes. :)

**Reply by andreypolyak** [reply]
Awesome! 👍🏻
